internal/config/config.go

```diff
@@ -299,3 +300,9 @@ func (c *Config) SetPartialFlushMinSpans(minSpans int, origin telemetry.Origin)
 	c.partialFlushMinSpans = minSpans
 	telemetry.RegisterAppConfig("DD_TRACE_PARTIAL_FLUSH_MIN_SPANS", minSpans, origin)
 }
+
+func (c *Config) DynamicInstrumentationEnabled() bool {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	return c.dynamicInstrumentationEnabled
+}
```

Comment:
Do we need a `SetDynamicInstrumentationEnabled` function too? If not, can we update the PR description w the reason why this shouldn't be configurable?

Reply:
I originally did not include it because there was no use for it. But another PR by the debugger team is adding a programmatic API for dynamic instrumentation, so I'll add the Set API now.